### DISCUSSION TITLE
Multiple Auth

### DISCUSSION THREAD

**Discussion by swiftalker** [filamentphp/filament#3376] [Help] [answered]
Hello,
So i have problem there, i want to make a admin and user separated by different model. Admin by Admin model and User with User Model, so my question is where can i setting the model for this? or i just set the guard only? but i have problem there, if i login in admin in parallel he touches the user section as well to login

**Comment by danharrin** [comment, marked as answer]
Yes, you set the guard, which is linked to a user provider, which is linked to your Admin model and tables

But from experience, I can assure you that it's easier to define roles instead of splitting up your user model, not just in Filament, in Laravel in general

**Reply by swiftalker** [reply]
Thank u for answering, it seems that there should be documentation about this, but in my case, it requires the existence of these 2 models to avoid conflicts between databases that have been arranged in such a way.
